### PR TITLE
Remove 0 change when shrinking the table

### PR DESCRIPTION
**Pull request by JulesDT**
If we want to do so, this will remove the lines where the change is == 0 when the table is shrunk

Relates to https://github.com/alex/nyt-2020-election-scraper/issues/91

### CODE REVIEW

File: battleground-state-changes.html.tmpl [hunk 1]
```diff
@@ -156,7 +156,17 @@
         const shrinkTablesFeature = 'shrunk';
         feature(shrinkTablesFeature, '#shrink_button', button => {
             button.html('Expand Tables');
-            $('.table').each(function() { $(this).find('tbody tr:gt(2)').hide(); });
+            $('.table').each(function(){
+                var countOfNonHiddenBlock = 0
+                $(this).find('tbody tr').each(function() {
+                    var value = parseInt(($(this).find('td:nth(4)').text()))
```
Comment (connorshea) [inline battleground-state-changes.html.tmpl]:
```suggestion
                    var value = parseInt(($(this).find('td:nth(4)').text()));
```

**Comment by bojanrajkovic** [comment]
Meh. I don't care about mixed semicolons, personally. :) LGTM!

**Comment by mr-ransel** [comment]
I'm not 100% certain that we'd want to, just since even when the change is 0, the Hurdle still is going to change based on the estimated votes remaining fluxuation

**Comment by thomasdesr** [comment]
Maybe we can reduce it to just keep most recent one.

**Review by thomasdesr — APPROVED** [review]

**Pull request merged**